packages/clarity-js/src/data/metadata.ts

```diff
@@ -71,17 +81,27 @@ export function consent(): void {
 }
 
 export function clear(): void {
-  // Clear any stored information in the session so we can restart fresh the next time
-  if (supported(window, Constant.SessionStorage)) { sessionStorage.removeItem(Constant.StorageKey); }
+  // Clear any stored information in the session cookie so we can restart fresh the next time
+  setCookie(Constant.SessionKey, Constant.Empty, 0);
+}
+
+function tab(): string {
+  let id = shortid();
+  if (supported(window, Constant.SessionStorage)) {
```

Comment:
>supported [](start = 6, length = 9)

should we have config.track check here?

Reply:
I thought tracking tab is not the same as tracking user or session. But, I do see your point. Let me add the check.

Reply:
Done